DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/CompetencyDataService.cs

```diff
@@ -70,12 +70,12 @@ public partial class SelfAssessmentDataService
                     COALESCE (rr.LevelRAG, 0) AS ResultRAG
                 FROM CandidateAssessments ca
                 INNER JOIN SelfAssessmentResults s
-                    ON s.CandidateID = ca.CandidateID AND s.SelfAssessmentID = ca.SelfAssessmentID
+                    ON s.CandidateID = ca.CandidateID_deprecated AND s.SelfAssessmentID = ca.SelfAssessmentID

```

Comment:
Please revert all of these SQL changes (that swap CandidateID for CandidateID_deprecated). We shouldn't be using the deprecated field. They have been addressed in TD-262 and these changes conflict with them.